cfpb/consumer-credit-trends
# WIP: deploy compiled HTML to gh-pages with Travis

Adds Travis config and restructures development workflow and release workflows to automate the charts being copied to gh-pages for review and deployment purposes.

To see the gh-pages branch on my fork go to here: https://github.com/cfarm/consumer-credit-trends/tree/gh-pages
the charts folder and the index file should be visible in the file tree there.

The index page is here: https://cfarm.github.io/consumer-credit-trends/

Currently fixing that so the filepaths are corrected for deployment (as well as working locally)

## Additions

-

## Removals

-

## Changes

-

## Testing

- delete your local `/charts/` folder in the root of the repo
- run `./setup.sh`
- run `gulp release`
- confirm your `charts` folder is now compiled to the `/dist/` folder
- confirm `gulp watch` still opens a functioning index.html page

## Review

- @user

[Preview this PR without the whitespace changes](?w=0)

## Screenshots


## Notes

-

## Todos

-

## Checklist
